zebra-scan/src/tests.rs

```diff
@@ -166,6 +168,118 @@ async fn scanning_from_fake_generated_blocks() -> Result<()> {
     Ok(())
 }
 
+/// Scan a populated state for the ZECpages viewing key.
+/// This test is very similar to `scanning_from_populated_zebra_state` but with the ZECpages key.
+/// There are no zechub transactions in the test data so we should get empty related transactions.
+#[tokio::test]
+async fn scanning_zecpages_from_populated_zebra_state() -> Result<()> {
+    /// The extended Sapling viewing key of [ZECpages](https://zecpages.com/boardinfo)
+    const ZECPAGES_VIEWING_KEY: &str = "zxviews1q0duytgcqqqqpqre26wkl45gvwwwd706xw608hucmvfalr759ejwf7qshjf5r9aa7323zulvz6plhttp5mltqcgs9t039cx2d09mgq05ts63n8u35hyv6h9nc9ctqqtue2u7cer2mqegunuulq2luhq3ywjcz35yyljewa4mgkgjzyfwh6fr6jd0dzd44ghk0nxdv2hnv4j5nxfwv24rwdmgllhe0p8568sgqt9ckt02v2kxf5ahtql6s0ltjpkckw8gtymxtxuu9gcr0swvz";
+
+    // Parse the key from ZECpages
+    let efvk = decode_extended_full_viewing_key(
+        HRP_SAPLING_EXTENDED_FULL_VIEWING_KEY,
+        ZECPAGES_VIEWING_KEY,
+    )
+    .unwrap();
+
+    let account = AccountId::from(1);
+
+    // Build a vector of viewing keys `vks` to scan for.
+    let fvk = efvk.fvk;
+    let ivk = fvk.vk.ivk();
+    let vks: Vec<(&AccountId, &SaplingIvk)> = vec![(&account, &ivk)];
+
+    let network = zebra_chain::parameters::Network::default();
```

Comment:
Nitpick: Since you're using a mainnet blocks constant below, it would be clearer to say `Mainnet` explicitly here.

```suggestion
    let network = zebra_chain::parameters::Network::Mainnet;
```